src/ft/mod.rs

```diff
@@ -155,6 +164,11 @@ impl Rasterize for FreeTypeRasterizer {
     }
 
     fn load_font(&mut self, desc: &FontDesc, size: Size) -> Result<FontKey, Error> {
+        if self.initialization_time_stamp.map_or(true, |t| t.elapsed() > RELOAD_DELAY) {
```

Comment:
Is there a reason why you decided to go without `.take()` here?

Reply:
Yes, it works only once with `take`, so if you call this function twice before `RELOAD_DELAY` it'll try to reload font, since time stamp is `None`, however I don't want to make it `None` until the time elapsed.

Reply:
Oh, duh. I didn't actually think that one through.